test/support/server/component-config.debug.json

```diff
@@ -0,0 +1,24 @@
+{
+    "../../../index" : {
+        "services": {
+            "remote.jb": {
+                "dataSource": "remote-service",
+                "discovery": {
+                    "models": {
+                        "Book": true,
+                        "Author": {
+                            "isPublic": true
+                        },
+                        "Publisher": false,
+                        "RemoteModel": {
+                            "isGlobal": false
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "loopback-component-explorer": {
```

Comment:
Do we need the explorer for the tests? I thought about it slowing down the boot process and therefore the app but I did not test that. Only have in mid that the parsing of the swagger.json takes a while when loading the explorer in the FE

Reply:
This is not used in the tests (see the environment).